packages/svelte/src/compiler/phases/scope.js

```diff
@@ -336,6 +342,124 @@ class Evaluation {
 				break;
 			}
 
+			case 'CallExpression':
+				{
+					const rune = get_rune(expression, scope);
+					if (
+						rune &&
+						scope.get(
+							object(/** @type {Identifier | MemberExpression} */ (expression.callee))?.name ?? ''
+						) === null
+					) {
```

Comment:
no need for the check here, it's taken care of inside `get_rune`

```suggestion
					if (rune) {
```